src/index.js

```diff
@@ -13,5 +13,9 @@ export default [
   'Writing tests takes x3 time than writing actual code.',
   'We need you to work on features. Not performance.',
   'Please do not worry a lot about quality. This is a prototype.',
+  'Since the tests dont pass, just skip them for this delivery build.',
```

Comment:
Suggestion to simplify (also added a missing apostrophe):

`Since tests don\'t pass, just skip them for this delivery build.`

Reply:
agree